docs/source/markdown/podman-run.1.md

```diff
@@ -231,6 +231,8 @@ Note: if the user only has access rights via a group, accessing the device
 from inside a rootless container will fail. The **crun**(1) runtime offers a
 workaround for this by adding the option **--annotation run.oci.keep_original_groups=1**.
 
+When running as root, Podman may load kernel modules required for using the specified device.
```

Comment:
IDK, but think we might want to list the modules we might load.  Same thought for create.  Maybe:
```suggestion
When running as root, Podman may load kernel modules required for using the specified device.  The modules that Podman will load when necessary are:  /dev/fuse
```

Reply:
This should be 
The devices that podman will load modules when necessary are: /dev/fuse

Reply:
I dropped "When running as root@ as the device is opened as rootless as well